corehq/apps/domain/views.py

```diff
@@ -1255,12 +1255,10 @@ def create_form(form_class):
     @property
     @memoized
     def select_subscription_type_form(self):
-        if self.request.method == 'POST':
-            for form_slug in self.slug_to_form:
```

Comment:
yeah, this is one where I wonder if it ever worked, given that it's validation on an internal page with few users, might not have gotten heavy qa and wouldn't often be tripped by accident